packages/waku/src/lib/plugins/vite-plugin-rsc-hmr.ts

```diff
@@ -126,16 +126,24 @@ export function rscHmrPlugin(): Plugin {
   };
 }
 
-const pendingMap = new WeakMap<ViteDevServer['ws'], Set<string>>();
+const pendingMap = new WeakMap<
+  ViteDevServer['hot'] | ViteDevServer['ws'],
```

Comment:
Now I'm thinking, can this be `ReturnType<typeof viteHot>`? what do you think @mjh316